package.json

```diff
@@ -5,14 +5,14 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "gatsby": "^4.0.0-next.1",
-    "gatsby-plugin-image": "^2.0.0-next.1",
-    "gatsby-plugin-manifest": "^4.0.0-next.1",
-    "gatsby-plugin-offline": "^5.0.0-next.1",
-    "gatsby-plugin-react-helmet": "^5.0.0-next.1",
-    "gatsby-plugin-sharp": "^4.0.0-next.1",
-    "gatsby-source-filesystem": "^4.0.0-next.1",
-    "gatsby-transformer-sharp": "^4.0.0-next.1",
+    "gatsby": "^4.0.0-zz-next.1",
+    "gatsby-plugin-image": "^2.0.0-zz-next.1",
+    "gatsby-plugin-manifest": "^4.0.0-zz-next.1",
+    "gatsby-plugin-offline": "^5.0.0-zz-next.1",
+    "gatsby-plugin-react-helmet": "^5.0.0-zz-next.1",
+    "gatsby-plugin-sharp": "^4.0.0-zz-next.1",
+    "gatsby-source-filesystem": "^4.0.0-zz-next.1",
+    "gatsby-transformer-sharp": "^4.0.0-zz-next.1",
```

Comment:
Are all of these available at these dependency versions? 🤔 

Furthermore, I understood our task as just updating the `gatsby` package. Not the plugins / source / transformers.

Reply:
Pulled this branch down and installed with no issues, so ignore my comment. 😄